edx-platform/pearson-vue-theme/lms/templates/header/navbar-logo-header.html

```diff
@@ -14,15 +14,13 @@
 %>
 
 <h1 class="header-logo">
-  <a href="${branding_api.get_home_url()}">
+  <a href="${configuration_helpers.get_value('LMS_ROOT_URL', branding_api.get_home_url())}">
```

Comment:
Is branding_api.get_home_url() not site-aware? @luismorenolopera

Reply:
is not site-aware